sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -552,6 +552,8 @@ object FunctionRegistry {
     expression[TimeWindow]("window"),
     expression[MakeDate]("make_date"),
     expression[MakeTimestamp]("make_timestamp"),
+    expression[MakeTimestampNTZ]("make_timestamp_ntz", true),
```

Comment:
If `setAlias` is set `true`, it means that an expression has multiple names but in your case `MakeTimestampNTZ` has only `make_timestamp_ntz`. Or `setAlias` has another meaning?

Reply:
It means we don't need to override the `prettyName` method.

Reply:
It's better to implement `prettyName`. The alias here is implemented via `TreeNodeTag`, which is quite unreliable.

Reply:
@beliefer can you fix it in a followup pr?

Reply:
@cloud-fan Thank you ping me.